cmd/ls.go

```diff
@@ -42,14 +42,14 @@ func printFolderMetadata(w io.Writer, e *files.FolderMetadata, longFormat bool)
 	if longFormat {
 		fmt.Fprintf(w, "-\t-\t-\t")
 	}
-	fmt.Fprintf(w, "%s\n", e.Name)
+	fmt.Fprintf(w, "%s\t\t", e.Name)
```

Comment:
FWIW, I don't think this is the most pressing problem for dbxcli right now :-) Feel free to work on other things (e.g. allowing cp/mv/rm of multiple files, supporting shell expansion etc).